app/src/app_desktop.cc

```diff
@@ -36,17 +38,30 @@
 namespace firebase {
 DEFINE_FIREBASE_VERSION_STRING(Firebase);
 
-namespace {
-
-// Size is arbitrary, just making sure that there is a sane limit.
-static const int kMaxBuffersize = 1024 * 500;
+namespace internal {
 
 // Attempts to load a config file from the path specified, and use it to
 // populate the AppOptions pointer.  Returns true on success, false otherwise.
-static bool LoadAppOptionsFromJsonConfigFile(const char* path,
-                                             AppOptions* options) {
+bool LoadAppOptionsFromJsonConfigFile(const char* path, AppOptions* options) {
+  // Size is arbitrary, just making sure that there is a sane limit.
+  static const int kMaxBuffersize = 1024 * 500;
+
   bool loaded_options = false;
+#if FIREBASE_PLATFORM_WINDOWS
```

Comment:
Note: For testing, if you change this line to "#if 0", the new unit tests fail (as expected). With it as-is, the tests pass.